carbon-design-system/carbon-addons-iot-react
# feat(table): add option to freeze header and footer

Closes #3696 

**Summary**

- Add prop to freeze header and footer

**Change List (commits, features, bugs, etc)**

- Add css class on table container level that sets `position: sticky` to table header
- Adjust table `height` property
- Add e2e test
- Update snapshots, docs

**Acceptance Test (how to verify the PR)**

- Go to [this story](https://deploy-preview-3754--carbon-addons-iot-react.netlify.app/?path=/story/1-watson-iot-table--with-pinned-header-and-footer)
- Scroll table
- Verify that table header and footer are pinned
- Click on filter icon to activate filter row
- Verify that table header is pinned with filter row

**Regression Test (how to make sure this PR doesn't break old functionality)**

- tests here



**Things to look for during review**
